--- webapp/graphite/local_settings.py.example	2012-05-31 10:28:54.000000000 +0400
+++ webapp/graphite/local_settings.py.example	2012-11-20 08:46:22.810797249 +0400

@@ -48,8 +48,8 @@
 # Most installs done outside of a separate tree such as /opt/graphite will only
 # need to change these three settings. Note that the default settings for each
 # of these is relative to GRAPHITE_ROOT
-#CONF_DIR = '/opt/graphite/conf'
-#STORAGE_DIR = '/opt/graphite/storage'
-#CONTENT_DIR = '/opt/graphite/webapp/content'
+CONF_DIR = '/etc/graphite-web'
+STORAGE_DIR = '/var/lib/carbon'
+CONTENT_DIR = '/usr/share/graphite-web/webapp/content'
 
 # To further or fully customize the paths, modify the following. Note that the
