--- server/core/session.c.old	2015-05-22 11:02:45.289256711 +0000
+++ server/core/session.c	2015-05-22 11:04:11.398014145 +0000
@@ -242,16 +242,16 @@
                 
 		if (session->client->user == NULL)
 		{
-			LOGIF(LT, (skygw_log_write(
-				LOGFILE_TRACE,
+			LOGIF(LM, (skygw_log_write(
+				LOGFILE_MESSAGE,
 				"Started session [%lu] for %s service ",
 				session->ses_id,
 				service->name)));
 		}
 		else
 		{
-			LOGIF(LT, (skygw_log_write(
-				LOGFILE_TRACE,
+			LOGIF(LM, (skygw_log_write(
+				LOGFILE_MESSAGE,
 				"Started %s client session [%lu] for '%s' from %s",
 				service->name,
 				session->ses_id,
