diff --git a/src/mongo/db/storage/mmap_v1/mmap_v1_engine.cpp b/src/mongo/db/storage/mmap_v1/mmap_v1_engine.cpp
index 5a784b25dc..96d1370d47 100644
--- a/src/mongo/db/storage/mmap_v1/mmap_v1_engine.cpp
+++ b/src/mongo/db/storage/mmap_v1/mmap_v1_engine.cpp
@@ -32,6 +32,10 @@
 
 #include "mongo/db/storage/mmap_v1/mmap_v1_engine.h"
 
+#if (__GLIBC_MINOR__ >= 25)
+#include <sys/sysmacros.h>
+#endif
+
 #include <boost/filesystem/operations.hpp>
 #include <boost/filesystem/path.hpp>
 #include <fstream>
